Module Path Resolution Error When Using Monorepo + Nitro + Prisma

### Environment

Linux x64, NodeJS v22.15.0

### Reproduction

I've created a minimal reproduction repo using pnpm monorepo + Nitro + Prisma, which reliably reproduces the bug on my machine (Linux x64, NodeJS v22.15.0):

[https://github.com/MagmaBlock/nitro-prisma-monorepo-debug](https://github.com/MagmaBlock/nitro-prisma-monorepo-debug)

### Describe the bug

In a pnpm monorepo environment, Nitro appears to have issues resolving module paths when importing local workspace packages.

The specific symptom is that in `.nitro/dev/index.mjs` (or `.nuxt/dev/index.mjs`), incorrect relative paths are being generated for the imported Prisma client:

```javascript
import { PrismaClient } from '../../../../../../../../../packages/database/generated/client';
```

This path contains an excessive number of `../`, clearly going beyond the valid directory structure. In VSCode, this path also cannot be clicked through or resolved.

Meanwhile, all other imports are using absolute paths like this:

```javascript
import consola from 'file:///home/magma/projects/nitro-prisma-monorepo-debug/node_modules/.pnpm/consola@3.4.2/node_modules/consola/dist/index.mjs';
import { ErrorParser } from 'file:///home/magma/projects/nitro-prisma-monorepo-debug/node_modules/.pnpm/youch-core@0.3.2/node_modules/youch-core/build/index.js';
import { Youch } from 'file:///home/magma/projects/nitro-prisma-monorepo-debug/node_modules/.pnpm/youch@4.1.0-beta.8/node_modules/youch/build/index.js';
import { SourceMapConsumer } from 'file:///home/magma/projects/nitro-prisma-monorepo-debug/node_modules/.pnpm/source-map@0.7.4/node_modules/source-map/source-map.js';
import { Server } from 'node:http';
```

Only the Prisma import uses a relative path — all others use absolute file URLs. This leads to runtime errors when trying to use the Prisma client inside Nitro:

```
Cannot find module '/packages/database/generated/client' imported from /home/magma/projects/nitro-prisma-monorepo-debug/apps/nitro/.nitro/dev/index.mjs
```

And the build also fails as a result.

### Additional context

_No response_

### Logs

```sh

```

--- a/packages/client-generator-ts/src/utils/wasm.ts
+++ b/packages/client-generator-ts/src/utils/wasm.ts
@@ -33,7 +33,6 @@ export function buildGetWasmModule({
   component,
   runtimeName,
   runtimeBase,
-  target,
   activeProvider,
   moduleFormat,
 }: BuildWasmModuleOptions) {
@@ -102,13 +101,11 @@ config.${component}Wasm = {
   }
 
   if (buildEdgeLoader) {
-    const fullWasmModulePath = target === 'vercel-edge' ? `${wasmModulePath}?module` : wasmModulePath
-
     return `config.${component}Wasm = {
   getRuntime: async () => await import(${JSON.stringify(wasmBindingsPath)}),
 
   getQuery${capitalizedComponent}WasmModule: async () => {
-    const { default: module } = await import(${JSON.stringify(fullWasmModulePath)})
+    const { default: module } = await import(${JSON.stringify(`${wasmModulePath}?module`)})
     return module
   }
 }`

--- a/packages/client-generator-ts/tests/utils/buildGetWasmModule.test.ts
+++ b/packages/client-generator-ts/tests/utils/buildGetWasmModule.test.ts
@@ -57,6 +57,11 @@ function makeTestCombinations() {
   for (const component of components) {
     for (const runtimeName of runtimeNames) {
       for (const target of targets) {
+        // Skip impossible combinations
+        if (['edge', 'wasm-compiler-edge'].includes(runtimeName) && !['vercel-edge', 'workerd'].includes(target)) {
+          continue
+        }
+
         for (const moduleFormat of moduleFormats) {
           const testName = `${component}-${runtimeName}-${target}-${moduleFormat}` as const
           combinations.push({

